### PR TITLE
TW-397 Doc Lightstep integration

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/lightstep.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    url: "/docs/sending-requests/intro-to-collections/"
+---
+
+Lightstep Instant Response is an incident response platform from ServiceNow, which offers the ability to observe, assess, resolve, and escalate alerts and issues in enterprise and DevOps applications. It's available at [lightstep.com](https://www.lightstep.com).
+
+ServiceNow has created a Postman integration for Lightstep. This integration ingests Postman monitoring alerts into Lightstep to give visibility and remediation guidance for ops and support teams. When a configured Postman monitor fails, an alert is sent to Lightstep. Postman's [webhooks integration](src/pages/docs/integrations/webhooks.md) is used to add the integration to Postman.
+
+To use Lightstep with Postman, use the Postman integration within Lightstep to create a webhook. Then, add a webhook integration in Postman and add the URL of the Lightstep webhook.
+
+See the [Lightstep documentation for the Postman integration](https://lightstep.com/incident-response/docs/integrations-postman) for a complete procedure on how to add a Postman integration to Lightstep.
```
Comment (Andy-GetPostman) [inline src/pages/docs/integrations/available-integrations/lightstep.md:19]: See the [Lightstep documentation](https://lightstep.com/incident-response/docs/integrations-postman) for a step-by-step guide on how to add the Postman integration.

File: src/pages/docs/integrations/available-integrations/lightstep.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+search_keyword: "lightstep, servicenow"
+contextual_links:
+  - type: section
+    name: "Prerequisites"
+  - type: link
+    name: "Grouping requests in collections"
+    url: "/docs/sending-requests/intro-to-collections/"
+---
+
+Lightstep Instant Response is an incident response platform from ServiceNow, which offers the ability to observe, assess, resolve, and escalate alerts and issues in enterprise and DevOps applications. It's available at [lightstep.com](https://www.lightstep.com).
```
Comment (mark-anthony-postman) [inline src/pages/docs/integrations/available-integrations/lightstep.md:13]:
```suggestion
[Lightstep Incident Response](https://lightstep.com/incident-response) is an incident response platform from ServiceNow, which offers the ability to observe, assess, resolve, and escalate alerts and issues in enterprise and DevOps applications.
```

File: src/pages/docs/integrations/available-integrations/lightstep.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+ServiceNow has created a Postman integration for Lightstep. This integration ingests Postman monitoring alerts into Lightstep to give visibility and remediation guidance for ops and support teams. When a configured Postman monitor fails, an alert is sent to Lightstep. Postman's [webhooks integration](src/pages/docs/integrations/webhooks.md) is used to add the integration to Postman.
```
Comment (mark-anthony-postman) [inline src/pages/docs/integrations/available-integrations/lightstep.md:15]:
```suggestion
ServiceNow has created a Postman integration for Lightstep. This integration ingests Postman monitoring alerts into Lightstep to give visibility and remediation guidance for ops and support teams. When a configured Postman monitor fails, an alert is sent to Lightstep. Postman's [webhooks integration](/docs/integrations/webhooks/) is used to add the integration to Postman.
```

**Review by mark-anthony-postman — APPROVED** [review]
LeftNavItems.jsx was missing a comma--I pushed a fix so I could run the dev site
Made a couple of additional comments, but otherwise LGTM!

**Pull request merged**